projects/plugins/protect/src/js/components/footer/index.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Container, Col, Text, Button } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { Icon, bug, cloud, shield } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import styles from './styles.module.scss';
+
+const Footer = () => {
+	return (
+		<Container horizontalSpacing={ 3 } horizontalGap={ 3 } fluid={ false }>
+			<Col sm={ 4 } md={ 8 } lg={ 12 }>
+				<Icon icon={ cloud } size={ 18 } />
+				<Icon icon={ shield } size={ 18 } />
+				<Icon icon={ bug } size={ 18 } />
+			</Col>
+			<Col sm={ 4 } md={ 4 } lg={ 6 }>
+				<Text variant="title-medium" className={ styles.titles }>
+					{ __( 'Comprehensive Site Security', 'jetpack-protect' ) }
+				</Text>
```

Comment:
I think we should use `<Title />` here

```suggestion
				<Title className={ styles.titles }>
					{ __( 'Comprehensive Site Security', 'jetpack-protect' ) }
				</Title>
```